charts/linkerd-crds/Chart.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   - name: partials
     version: 0.1.0
     repository: file://../partials
-version: 1.9.2-edge
+version: 1.9.3-edge
```

Comment:
Was there a change to any CRDs in this release?

Reply:
Not in this one but in the previous one, but this version didn't get bumped.